assets/css/style.css

```diff
@@ -81,16 +70,177 @@ nav a {
 
 .hero .social_link {
   margin-top: 10px;
-  height: 20px;
-  width: 170px;
-  left: 24px;
-  top: 470px;
 }
 
 .social_link a {
   color: #505f79;
-  height: 19px;
-  width: 19px;
-  left: 24px;
-  top: 470px;
+}
+
+/* Work Section */
+.work-section {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  grid-gap: 40px;
+  padding: 24px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
## Styling ✨✨

- [ ] Kindly, it would be better to replicate the given ```font-colors, font-size, and the spacing``` of your elements as given by the Figma design 😎😎

   Replicating the font `colors, sizes, and alignments` from the Figma design will help ensure that the final product looks consistent and polished.  😎😎
   

- [ ] It would be better also to replicate as much as possible the Figma design's `button` on your project and this will help give the overall design a more cohesive look and feel. 👍 👍 

### Kindly check the following comparison 👇 
![image](https://user-images.githubusercontent.com/102757126/236477368-180e3716-0a02-46be-a5bc-90c6f2afeb7b.png)